src/managers/AnimationManager.cpp

```diff
@@ -245,10 +236,7 @@ void CAnimationManager::tick() {
                                            BORDERSIZE + ROUNDINGSIZE); // bottom
 
                 // damage for new box
-                CBox       WLRBOXNEW        = {PWINDOW->m_vRealPosition.value(), PWINDOW->m_vRealSize.value()};
-                const auto PWINDOWWORKSPACE = g_pCompositor->getWorkspaceByID(PWINDOW->m_iWorkspaceID);
-                if (PWINDOWWORKSPACE)
-                    WLRBOXNEW.translate(PWINDOWWORKSPACE->m_vRenderOffset.value());
+                CBox WLRBOXNEW = {PWINDOW->m_vRealPosition.value(), PWINDOW->m_vRealSize.value()};
```

Comment:
isnt there a util to get this? plus, shouldn't we add the workspace offset?

Reply:
Yeah I'll add the offset back. Problem is I noticed if you do that it will add a little damage to other monitors when you switch workspaces. But maybe it should be properly fixed later.

Reply:
I guess we can add an optional monitor param to `addDamage`?

Reply:
up to you if you want to do this in this mr, ping me with your preferred solution

Reply:
@vaxerski Probably better to merge this one first. I can draft up something to fix that issue later

Reply:
oke